Make GIA demos consistent with Mantle and tidy up GPlates case a little

drhodrid opened the pull request:
Attempt 2!!

Patol75 commented on demos/glacial_isostatic_adjustment/2d_cylindrical/2d_cylindrical.py:
```diff
@@ -48,8 +49,8 @@
 # Set up function spaces - currently using the bilinear Q2Q1 element pair:
-V = VectorFunctionSpace(mesh, "Q", 2)  # (Incremental) Displacement function space (vector)
-W = FunctionSpace(mesh, "Q", 1)  # Pressure function space (scalar)
+V = VectorFunctionSpace(mesh, "CG", 2)  # (Incremental) Displacement function space (vector)
+W = FunctionSpace(mesh, "CG", 1)  # Pressure function space (scalar)
 S = TensorFunctionSpace(mesh, "DQ", 2)  # (Discontinuous) Stress tensor function space (tensor)
```
Something I have noticed: Firedrake does not complain if you provide `CG` instead of `Q` for quadrilateral elements, but it will raise a warning if you ask for `DG` instead of `DQ` (`UserWarning: Discontinuous Lagrange element requested on quadrilateral, creating DQ element.`). So this raises the question of consistency differently: is it fine to have `CG` and `DQ` in the same script?

w3168 replied:
I noticed that too...

drhodrid replied:
I think it's fine, but for consistency, perhaps we specify DQ here as DG?

Patol75 reviewed (commented):
Regarding the comment changes, have you mostly limited the line length or also incorporated text changes? It seems to be the former, but I just want to make sure.

drhodrid commented:
Mostly line length, but also corrected a few typos.

On 13 Nov 2024, at 3:08 pm, Thomas Duvernay ***@***.***> wrote:



@Patol75 commented on this pull request.

Regarding the comment changes, have you mostly limited the line length or also incorporated text changes? It seems to be the former, but I just want to make sure.

—
Reply to this email directly, view it on GitHub<https://github.com/g-adopt/g-adopt/pull/146#pullrequestreview-2431729722>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AB25UKUTXS4PLL5MAUBY4MD2ALGE3AVCNFSM6AAAAABRVQ22ZOVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDIMZRG4ZDSNZSGI>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

drhodrid commented:
@Patol75 @w3168 - are you happy for this to go in?

sghelichkhani reviewed (approved):
LGTM! Just a note that it is probably easier to review this if we seperate changes like this into a PR of just cosmetic new-line changes, and one with actual changes in text. Keeps tracking of what changes happen when a lot easier.

w3168 commented on demos/glacial_isostatic_adjustment/base_case/base_case.py:
```diff
@@ -206,11 +248,14 @@
 # association of degrees of freedom with the mesh and pointing to the
-# finite element basis. We will choose Q2-Q1 for the mixed incremental displacement-pressure similar to our mantle convection demos. This is a Taylor-Hood element pair which has good properties for Stokes modelling. We also initilaise a discontinuous tensor function space that wil store our previous values of the deviatoric stress, as the gradient of the continous incremental displacement field w ...
+# finite element basis. We will choose Q2-Q1 for the mixed incremental displacement-pressure similar to our mantle convection demos.
+# This is a Taylor-Hood element pair which has good properties for Stokes modelling. We also initilaise a discontinuous tensor function
```
initialise

w3168 commented:
> @Patol75 @w3168 - are you happy for this to go in?

I spotted a small typo in initialise but apart from that looks good! thanks @drhodrid !
